cognite/client/_api/data_modeling/instances.py

```diff
@@ -638,13 +639,13 @@ def apply(
             "replace": replace,
         }
         nodes = nodes or []
-        nodes = nodes if isinstance(nodes, Sequence) else [nodes]
+        nodes = nodes if isinstance(nodes, Sequence) else [nodes]  # type: ignore[assignment]
 
         edges = edges or []
-        edges = edges if isinstance(edges, Sequence) else [edges]
+        edges = edges if isinstance(edges, Sequence) else [edges]  # type: ignore[assignment]
 
         res = self._create_multiple(
-            items=cast(Sequence[WriteableCogniteResource], (*nodes, *edges)),
+            items=cast(Sequence[WriteableCogniteResource], (*nodes, *edges)),  # type: ignore[misc]
```

Comment:
Hmm. Should TypedInstanceWrite actually inherit from `WriteableCogniteResource` instead of `CogniteResource`?

Reply:
No, I don't think so. Writable is for the read version of the resource.